cmd/sebak/cmd/wallet/payment.go

```diff
@@ -0,0 +1,223 @@
+)
+
+var (
+	flagNetworkID     string = sebakcommon.GetENVValue("SEBAK_NETWORK_ID", "")
+	PaymentCmd        *cobra.Command
+	flagEndpoint      string
+	flagCreateAccount bool
+)
+
+func init() {
+	PaymentCmd = &cobra.Command{
+		Use:   "payment <receiver pubkey> <amount> <sender private key>",
+		Short: "Send <amount> BOSCoin from one wallet to another",
+		Args:  cobra.ExactArgs(3),
+		Run: func(c *cobra.Command, args []string) {
+			var err error
+			var amount sebak.Amount
+			var sender keypair.KP
+			var receiver keypair.KP
+			var endpoint *sebakcommon.Endpoint
+
+			// Receiver's public key
+			if receiver, err = keypair.Parse(args[0]); err != nil {
+				common.PrintFlagsError(c, "<receiver public key>", err)
+			} else if _, err = receiver.Sign([]byte("witness")); err == nil {
+				common.PrintFlagsError(c, "<receiver public key>", fmt.Errorf("Provided key is a private key, not an address"))
+			}
+
+			// Amount
+			if amount, err = common.ParseAmountFromString(args[1]); err != nil {
+				common.PrintFlagsError(c, "<amount>", err)
+			}
+
+			// Sender's private key
+			if sender, err = keypair.Parse(args[2]); err != nil {
+				common.PrintFlagsError(c, "<sender private key>", err)
+			} else if _, ok := sender.(*keypair.Full); !ok {
+				common.PrintFlagsError(c, "<sender private key>", fmt.Errorf("Provided key is an address, not a private key"))
+			}
+
+			// Check a network ID was provided
+			if len(flagNetworkID) == 0 {
+				common.PrintFlagsError(c, "--network-id", fmt.Errorf("A --network-id needs to be provided"))
+			}
+
+			if endpoint, err = sebakcommon.ParseNodeEndpoint(flagEndpoint); err != nil {
+				common.PrintFlagsError(c, "--endpoint", err)
+			}
+
+			// TODO: Validate input transaction (does the sender have enough money?)
+
+			// At the moment this is a rather crude implementation: There is no support for pooling of transaction,
+			// 1 operation == 1 transaction
+			var tx sebak.Transaction
+			var connection *sebakcommon.HTTP2Client
+			var sender_account sebak.BlockAccount
+
+			// Keep-alive ignores timeout/idle timeout
+			if connection, err = sebakcommon.NewHTTP2Client(0, 0, true); err != nil {
+				log.Fatal("Error while creating network client: ", err)
+				os.Exit(1)
+			}
+			client := sebaknetwork.NewHTTP2NetworkClient(endpoint, connection)
+
+			if sender_account, err = getSenderDetails(client, sender); err != nil {
+				log.Fatal("Could not fetch sender account: ", err)
+				os.Exit(1)
+			}
+
+			if flagCreateAccount {
+				tx = makeTransactionCreateAccount(sender, receiver, amount, sender_account.Checkpoint)
+			} else {
+				tx = makeTransactionPayment(sender, receiver, amount, sender_account.Checkpoint)
+			}
+
+			tx.Sign(sender, []byte(flagNetworkID))
+
+			// Send request
+			var retbody []byte
+			if retbody, err = client.SendMessage(tx); err != nil {
+				log.Fatal("Network error: ", err, " body: ", retbody)
+				os.Exit(1)
+			}
+			if len(retbody) != 0 {
+				log.Print(string(retbody))
```

Comment:
It's minor.  but `log.Print`  makes it difficult to use ` | jq . `.  What's your take on  using `fmt.Print` insteads of `log.Print`? :-> 

```
$docker run -it --rm --net host sebak wallet payment 10 SBECGI3FSCYHNQIMANNCWQSVA6S5C6L4BXFKAPMBAMI5V47NWXNE37MN GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW --endpoint=https://127.0.0.1:2822 --create | jq .
parse error: Invalid numeric literal at line 1, column 6
```

```
$docker run --rm --network host -it sebak wallet payment GDTEPFWEITKFHSUO44NQABY2XHRBBH2UBVGJ2ZJPDREIOL2F6RAEBJE4 10000000000 SBECGI3FSCYHNQIMANNCWQSVA6S5C6L4BXFKAPMBAMI5V47NWXNE37MN --endpoint=https://127.0.0.1:2822 --create
2018/06/29 03:39:57 {"T":"transaction","H":{"version":"","created":"2018-06-29T03:39:57.618528900Z","hash":"CvrDmmXV8YcQV3tr1vvPRT1UczajsZzE6ZhAM3kDfU5a","signature":"3TYE7GJL6Ts57PboiMnrNmLsKxiUPz46Xk8nehwTnuLxfM1vkk9umjopcqTQToc2VsDFhVWXFNypjbhxKxtDizw4"},"B":{"source":"GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ","fee":"10000","checkpoint":"5nc4PvPKMxDTM6ets3hudktkA-5nc4PvPKMxDTM6ets3hudktkA","operations":[{"H":{"type":"create-account"},"B":{"target":"GDTEPFWEITKFHSUO44NQABY2XHRBBH2UBVGJ2ZJPDREIOL2F6RAEBJE4","amount":"10000000000"}}]}}
```

Reply:
That makes sense, although I'm considering simply removing this print, since it just echoes back what we sent...